.github/workflows/review.yml

```diff
@@ -27,19 +27,9 @@ jobs:
       id: review-check-result
       run: |
         script_result=$(bash bin/github-review.sh ${PR_NUMBER} | tail -1)
```

Comment:
This will fail for non-deployment related PRs (which happen) - we should make sure that this check is ONLY run on deployment related PRs.

Reply:
A check based on the deployment file path has been added. It will fail for this one as there are changes made in this.

Reply:
Is there any advantage to doing it that way instead of using `path` for the triggers?

https://github.com/safe-global/safe-modules/blob/c403aff29bed6e3e326ab36c820094f2ada41c23/.github/workflows/ci_recovery.yml#L2-L5

Maybe not supported for `pull_request` triggers?

Reply:
I just checked, it does support `pull_request` triggers. Updated 👍🏾